localstack/services/events/events_listener.py

```diff
@@ -22,27 +21,36 @@ def forward_request(self, method, path, data, headers):
         action = headers.get('X-Amz-Target')
         if method == 'POST' and path == '/':
             parsed_data = json.loads(to_str(data))
-            if action == 'AWSEvents.PutEvents':
-                events_with_added_uuid = list(
-                    map(lambda event: {'event': event, 'uuid': str(uuid.uuid4())}, parsed_data['Entries']))
-                content = {'Entries': list(map(lambda event: {'EventId': event['uuid']}, events_with_added_uuid))}
-                self._create_and_register_temp_dir()
-                self._dump_events_to_files(events_with_added_uuid)
-                return make_response(content)
-            elif action == 'AWSEvents.ListTagsForResource':
-                return make_response(self.svc.list_tags_for_resource(
-                    parsed_data['ResourceARN']))
+
+            # if action == 'AWSEvents.PutEvents':
+            #     events_with_added_uuid = list(
+            #         map(lambda event: {'event': event, 'uuid': str(uuid.uuid4())}, parsed_data['Entries'])
+            #     )
+            #
+            #     self._create_and_register_temp_dir()
+            #     self._dump_events_to_files(events_with_added_uuid)
```

Comment:
Wondering if we should keep the `_dump_events_to_files(..)` logic for now, as some users are depending on it.

Instead of intercepting the request and generating UUIDs here, could we intercept the response - i.e., move this code block into `return_response(...)` further below, and then do something like:
```
def return_response(self, method, path, data, headers, response, request_handler=None):
    ...
    if action == 'AWSEvents.PutEvents':
        events = json.loads(to_str(response.content))['Entries']
        self._create_and_register_temp_dir()
        self._dump_events_to_files(events)
    ...
```